### ISSUE TITLE
third-party-import

### ISSUE THREAD

**Issue by Hexagon** [locale-kit/locale-kit-fresh#1]
Hello! Just replaced freshlate with this library, works like a charm. Thanks!

One suggestion - I did notice that the readme doesn't mention that the "consumer" need to add locale-kit to their import_map.json.

```jsonc
{
    "localekit_fresh": "https://deno.land/x/localekit_fresh@0.1.0/mod.ts",
    "locale-kit": "https://deno.land/x/localekit/mod.ts",
    // ...
}
```

Without it, they'll get

![image](https://user-images.githubusercontent.com/419737/210156249-aa34dc04-8d14-4610-ade6-76191c0e3ee3.png)

**Comment by brocococonut** [comment]
Strange...
I moved the import to that file as it only get's used once - no real point having it in the import_map if it's just going to cause issues. Did this fix it for you? (commit [7cf111e62ed07d6551770d95854ac12839ba410a](https://github.com/locale-kit/locale-kit-fresh/commit/7cf111e62ed07d6551770d95854ac12839ba410a))

I'll go ahead and create a release for it as well so you can test using that

**Comment by Hexagon** [comment]
Yeah, import_map only seem to be read once (by the base project) in deno. 

The problem is resolved now, thanks again :)
